pkg/mcproducer/testdata/machineconfig-test.yaml

```diff
@@ -21,6 +21,7 @@ spec:
             Type=oneshot
             TimeoutSec=10
             EnvironmentFile=/etc/mco/proxy.env
+            Environment="FIRMWARE_FILES_PATH=/tmp/test"
```

Comment:
I would use a path that is not in the `/tmp` directory. 
Is this working with `/tmp`? I think that everything in `/tmp` in the container is going to be overwritten by the mounted directory isn't it?

Reply:
The unit test will pass but you're right, I will change it to a different path to make it consistent.